source/Handlebars/IO/HtmlEncoder.cs

```diff
@@ -12,6 +12,13 @@ namespace HandlebarsDotNet
     /// </summary>
     public class HtmlEncoder : ITextEncoder
     {
+        private readonly Compatibility _compatibility;
+
+        public HtmlEncoder(Compatibility compatibility)
+        {
+            _compatibility = compatibility;
```

Comment:
I'd prefer having different encoder implementation.
This approach has at least 2 benefits: 
- easier to deprecate in the future by just removing old implementation
- performance benefit as no `if` would be involved at the time of rendering

Reply:
Pushed new commit that uses different encoder implementations (replaces bool feature toggle).
Something like this you were thinking of?

I'm delaying change to README until the implementation is confirmed.

Reply:
Yes, that's what I was thinking about 👍